src/lib/serviceRoutinesV2/postEntities.cpp

```diff
@@ -78,10 +83,19 @@ std::string postEntities
     ciP->httpHeaderValue.push_back(location);
     ciP->httpStatusCode = SccCreated;
   }
+  else if (rcode == SccInvalidModification)
+  {
+    OrionError oe;
+    oe.code = SccContextElementNotFound;
+    oe.reasonPhrase ="AlredyExists";
+    oe.details = "Entity alredy exists";
```

Comment:
I use OrionError because is used in src/lib/serviceRoutinesV2/deleteEntity.cpp but if its ErrorCode is a better class for this task, I can chenge it.
The SccContextElementNotFound is my mistake, sorry.